maven_iuvs/miscellaneous.py

```diff
@@ -51,3 +51,34 @@ def mirror_dn_to_deg(dn, inverse=False):
 
     # return the conversion
     return value
+
+
+def find_nearest(array, value):
+    """
+    Find the closest entry in array to value. 
+    """
+    array = np.asarray(array)
+    idx = (np.abs(array - value)).argmin()
+    return idx, array[idx]
+
+
+def get_grad_colors(L, cmap, strt=0, stp=1, mikes=False):
```

Comment:
should be in graphics

Reply:
Fixed by 6467166340944ca6a69510b1a6725e519a1a501e